packages/f36-docs-utils/src/Props/Props.tsx

```diff
@@ -0,0 +1,106 @@
+import React from 'react';
+import {
+  Table,
+  TableHead,
+  TableBody,
+  TableRow,
+  TableCell,
+  Flex,
+  Box,
+  TextLink,
+  Tooltip,
+} from '@contentful/f36-components';
+import type { PropType } from './types';
+import { useProps } from './PropsProvider';
+
+function EnumPropertyType({ type }: { type: PropType }) {
+  return (
+    <Flex flexWrap="wrap">
+      {type.value.map((value) => {
+        return (
+          <Box key={value.value} marginRight="spacingS" marginBottom="spacingS">
+            {value.value}
+          </Box>
+        );
+      })}
+    </Flex>
+  );
+}
+
+function LookupPropertyType({ type }: { type: PropType }) {
+  const [visible, setVisible] = React.useState(false);
+  if (visible) {
+    return <EnumPropertyType type={type} />;
+  }
+  return (
+    <Tooltip place="top" content="Click to lookup">
+      <TextLink
+        onClick={() => {
+          setVisible(true);
+        }}
+      >
+        {type.raw}
+      </TextLink>
+    </Tooltip>
+  );
+}
+
+function PropertyType({ type, name }: { type: PropType; name: string }) {
+  if (name === 'as') {
+    return (
+      <>
+        HTML Tag or React Component, e.g. <code>div</code>, <code>span</code>,
+        etc.
+      </>
+    );
+  }
+  if (name === 'children') {
+    return <>{type.raw}</>;
+  }
+  if (type.name === 'enum') {
+    if (type.value.length < 4) {
+      return <EnumPropertyType type={type} />;
+    }
+    return <LookupPropertyType type={type} />;
```

Comment:
Why have a lookup component instead of always just rendering the props?

Reply:
a lot of them have a lot of options... like a lot, not sure if we want to show them all the time by default.

Reply:
Do they fit in the current UI?

Reply:
yep, they fit but it can be a really long list. IMO, collapsing them in a lookup makes sense but I'm happy to discuss the potential change later.

Reply:
I think it's fine as is, I was just thinking based on the GIF that a long list might not work in the table at all.